rust_src/src/windows.rs

```diff
@@ -26,8 +26,8 @@ use crate::{
         windows_or_buffers_changed, wset_redisplay,
     },
     remacs_sys::{
-        face_id, glyph_matrix, glyph_row, pvec_type, vertical_scroll_bar_type, EmacsInt, Lisp_Type,
-        Lisp_Window,
+        face_id, glyph_matrix, glyph_row, pvec_type, vertical_scroll_bar_type, EmacsDouble,
+        EmacsInt, Lisp_Type, Lisp_Window,
```

Comment:
We usually try to import types separately from functions. This is all done to cut down on merge conflicts.

Reply:
I think these are types, but I separated the remacs binding ypes from the c struct types